Cargo.toml

```diff
@@ -12,7 +12,8 @@ keywords = ["web", "http", "protocol"]
 [features]
 default = ["server"]
```

Comment:
I would just include `threadpool` here also by default, since most of use cases will be using a threaded server.